pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/util/internal/pex/ThinPexGenerator.java

```diff
@@ -44,7 +44,7 @@
     public ThinPexGenerator(Project project, String entryPointTemplate, Map<String, String> extraProperties) {
         this.project = project;
         this.entryPointTemplate = entryPointTemplate;
-        this.extraProperties = extraProperties == null ? new HashMap<>() : extraProperties;
+        this.extraProperties = extraProperties == null ? new HashMap<String, String>() : extraProperties;
```

Comment:
Without the types I get the following error compiling
`error: incompatible types: Map<? extends Object,? extends Object> cannot be converted to Map<String,String>`. 

This has something to do with the turnary statement, if i convert it to 
```
if(null == extraProperties) {
    this.extraProperties = new HashMap<>();
} else {
    this.extraProperties = extraProperties;
}
```
that works, do you think the expanded format is more usable/readable/perferable?